templates/CRM/Mailing/Form/InsertTokens.tpl

```diff
@@ -140,7 +140,7 @@ function selectValue( val, prefix) {
   var dataUrl = {/literal}"{crmURL p='civicrm/ajax/template' h=0 }"{literal};
 
   cj.post( dataUrl, {tid: val}, function( data ) {
-    var hide = (data.document_path && isPDF) ? false : true;
+    var hide = (data.document_body && isPDF) ? false : true;
```

Comment:
Bcoz I renamed it to document_body here https://github.com/civicrm/civicrm-core/pull/8474/files#diff-ed1056b02289bcdce4d67408f8e21253R60